lib/cisco_node_utils/interface.rb

```diff
@@ -1026,11 +1026,12 @@ def switchport_vtp_mode_capable?
     end
 
     def switchport_vtp
-      return nil unless switchport_vtp_mode_capable?
+      return default_switchport_vtp unless switchport_vtp_mode_capable?
```

Comment:
@glennmatthews 

I know you suggested changing the default_switchport_vtp method so that it returns nil when switchport_vtp_mode_capable? is false, but I'm not sure that's the right way to go.  Shouldn't I be able to tell the default by looking in the yaml?  Willing to be convinced, however.